### PR TITLE
Fixed incorrect type-hint for parameter of type class-string.

### PR DESCRIPTION
**Pull request by thirsch**
I stumbled upon this a few times but never fixed it. Today is the day. ;-)

### CODE REVIEW

**Review by alquerci — APPROVED** [review]

**Review by thePanz — APPROVED** [review]
Thanks! 
Please rebase on latest `master` branch and I will merge it :+1: 
note: this repo's config enforces a linear git history

**Comment by thirsch** [comment]
Thanks! Rebased. :)

File: lib/config/sfConfigCache.class.php [hunk 1]
```diff
@@ -157,9 +157,9 @@ public function import($config, $once = true, $optional = false)
     /**
      * Registers a configuration handler.
      *
-     * @param string $handler The handler to use when parsing a configuration file
-     * @param class  $class   A configuration handler class
-     * @param string $params  An array of options for the handler class initialization
+     * @param string       $handler The handler to use when parsing a configuration file
+     * @param class-string $class   A configuration handler class
+     * @param string       $params  An array of options for the handler class initialization
```
Comment (thePanz) [inline lib/config/sfConfigCache.class.php:162]: should this be string[] instead?

File: lib/config/sfConfigCache.class.php [hunk 2]
```diff
@@ -157,9 +157,9 @@ public function import($config, $once = true, $optional = false)
     /**
      * Registers a configuration handler.
      *
-     * @param string $handler The handler to use when parsing a configuration file
-     * @param class  $class   A configuration handler class
-     * @param string $params  An array of options for the handler class initialization
+     * @param string       $handler The handler to use when parsing a configuration file
+     * @param class-string $class   A configuration handler class
```
Comment (thePanz) [inline lib/config/sfConfigCache.class.php:161]: we could be more specific here with `class-string<sfConfigHandler>`?

**Comment by thirsch** [comment]
Thanks for both suggestions. Unfortunately, there are so many errors in the type-hints.

**Comment by thePanz** [comment]
> Unfortunately, there are so many errors in the type-hints.

yes, true! Thank you for taking the first steps to clean them up! :+1:

**Pull request merged**